### DISCUSSION TITLE
ARM64 support

### DISCUSSION THREAD

**Discussion by roberto-tenil** [devlikeapro/waha#174] [Ideas]
WAHA is a great project.

I suggest that an update comes to make it work on the arm64 architecture.

`ERROR: no matching manifest for linux/arm64/v8 in the manifest list entries`

Hugs

**Comment by allburov** [comment]
Hi! Try one of these images, with `arm` tag
```
# Download the image
docker pull devlikeapro/whatsapp-http-api:arm
# Rename it, so you can use devlikeapro/whatsapp-http-api image in other place
docker tag devlikeapro/whatsapp-http-api:arm devlikeapro/whatsapp-http-api

# WAHA Plus
docker pull devlikeapro/whatsapp-http-api-plus:arm
```
https://waha.devlike.pro/docs/overview/quick-start/#11-download-image-on-arm

**Reply by roberto-tenil** [reply]
Wow, you are amazing.

Worked perfectly. Thanks a lot for the help.
